src/ape_accounts/accounts.py

```diff
@@ -92,6 +93,17 @@ def __key(self) -> HexBytes:
 
         return key
 
+    @property
+    def public_key(self) -> HexBytes:
+        key = self.__key
+
+        # Derive the public key from the private key
+        pk = keys.PrivateKey(key)
```

Comment:
could we save this to the account's JSON so we don't have to decrypt it like this? it is odd to require a private key to get a public key. for accounts that dont yet have it in the JSON, we could do this approach here and the save it in the JSON so next time decrypting is no required.